desc/objectives/_stability.py

```diff
@@ -422,15 +420,23 @@ def __init__(
         if target is None and bounds is None:
             target = 0
 
-        self._rho = rho
-        self._alpha = alpha
         self._nturns = nturns
         self._nzetaperturn = nzetaperturn
-        self._zeta0 = zeta0
         self._Neigvals = Neigvals
         self._lambda0 = lambda0
         self._w0 = w0
         self._w1 = w1
+        self._rho = rho
+        self._add_lcfs = np.all(self._rho < 0.97)
```

Comment:
why 0.97?

Reply:
I don't think there is a use-case for this objective with rho=1. The extrapolation should work fine when it is that close

Reply:
I guess this extrapolation is to calculate normalizing factors a_N and B_N. The solver itself is fine for all rhos <= 1.0